src/server/controllers/participants.js

```diff
@@ -103,15 +104,30 @@ exports.getDetails = async (req, res) => {
 // Add new participant
 exports.post = async (req, res) => {
   try {
-    const { participantdata } = req.body;
+    // check if user enter just one file
+    if (req.files.length >= 2) {
+      return res.status(302).send('The uploaded files should be just one file');
```

Comment:
I think this should be a 400 code - a [**bad request**](https://httpstatuses.com/400).
You should be able to to deal with the "redirect" on the client side.